spec/requests/historical_banner_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe "Documents" do
+  it "shows the history mode banner when the edition is in history mode" do
+    @edition = create(:edition, :published, :political, :past_government)
+    get document_path(@edition.document)
+    expect(response.body).to include(I18n.t!("documents.show.historical.title", document_type: @edition.document_type.label.downcase))
+    expect(response.body).to include(I18n.t!("documents.show.historical.description", government_name: @edition.government.name))
+  end
+
+  it "won't show the history mode banner when the edition was created under the current government" do
+    @edition = create(:edition, :published, :political, :current_government)
+    get document_path(@edition.document)
+    expect(response.body).not_to include(I18n.t!("documents.show.historical.title", document_type: @edition.document_type.label.downcase))
+    expect(response.body).not_to include(I18n.t!("documents.show.historical.description", government_name: @edition.government.name))
+  end
+
+  it "won't show the history mode banner when the edition is not political" do
+    @edition = create(:edition, :published, :not_political)
+    get document_path(@edition.document)
+    expect(response.body).not_to include(I18n.t!("documents.show.historical.title", document_type: @edition.document_type.label.downcase))
+  end
```

Comment:
Are we missing a test for not showing history mode banner if user does not have the prerelease permission?

Reply:
Our precedent is to avoid testing whether this is off and on as it gets pretty messy and the logic for pre-release features is usually trivial. The test user is assumed to have pre-release permissions.

Reply:
Ok, cheers! 👍

Reply:
There's prior art on us not doing this. I assume because it will only be removed later. 
https://github.com/alphagov/content-publisher/pull/1251/files